news/2017-12-13-qat-gazette-december-2017.md

```diff
@@ -0,0 +1,85 @@
+
+### New Beatmap Nominator Promotions
+
+In conjunction with the above changes, it is my great pleasure to once again welcome **new Beatmap Nominators for osu!taiko, osu!catch and osu!mania.**
+
+**For osu!taiko**: [Aloda](https://osu.ppy.sh/users/1190127)
+
+**For osu!catch**: [Hareimu](https://osu.ppy.sh/users/4138746), [Nokashi](https://osu.ppy.sh/users/5431196) and [Sorcerer](https://osu.ppy.sh/users/1913190)
+
+**For osu!mania**: [Garalulu](https://osu.ppy.sh/users/757783), [Tofu1222](https://osu.ppy.sh/users/6089608) and the returning [Kamikaze](https://osu.ppy.sh/users/2124783)
+
+I'm sure they are eager to deliver us all more quality content, so please offer them all your congratulations and support through their probation period and beyond! I look forward to working with you all in the near future!
+
+Additionally, congratulations to [celerih](https://osu.ppy.sh/users/4696296) and [Sinnoh](https://osu.ppy.sh/users/4236057) in **adding proficiency in osu!catch to their existing osu! Beatmap Nominator memberships!** As a former osu! Beatmap Nominator who branched into osu!catch in late 2015 myself, it's always great to see people continuing to expand their knowledge and tackling new challenges!
+
+## Within the Quality Assurance Team
+
+In addition to developing the Beatmap Nominator team and moderating the new features of Modding Discussions, the QAT will be seeking to grow existing features, ensuring that current modding systems keep pace, and that a future full transition over to new system is as smooth as possible.
+
+### Call for Metadata Helpers
+
+I mentioned last month that we **opened a Discord server to discuss and catalogue metadata across newly Qualified maps**. The idea behind this was to provide some assistance to the many of us who are not particularly experienced with metadata, but also to allow mappers and modders to assist each other. Users can additionally browse the existing catalogue, to see if their song/map has already been discussed and metadata confirmed before.
+
+Anyone is welcome to join the [Metadata Heap server](https://discord.gg/9Y4EdyM) if the above seems useful to you, but we’re especially interested in recruiting some dedicated and knowledgeable Helpers to assist our current team of experts. More details can be found on the server’s #announcements channel, but if you’re already involved there, head straight on over to the [application form](https://goo.gl/87mEQA) and show your interest!
+
+### Modifications to the beatmap Hype system
+
+Anyone who has participated in the Modding Discussions system will have noticed the “Hype Train” feature - billed as a replacement for kudosu star shooting from the previous forum modding system. Since it’s not tied to kudosu, however, Hype offers an opportunity for players to get involved in the promotion of beatmaps, without the need to necessarily mod them.
```

Comment:
`“` and `”` -> `"`
(this also occurrences for the paragraph below)

Reply:
Once again, not sure why these morphed into the curlier variety, but fixed